dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -274,7 +274,7 @@ private String getRSS() {
         schema = @Schema(implementation = String.class)))
     @ApiOperation(value = "Returns the file containing runner dependencies.", response = String.class)
     public Response getRunnerDependencies(
-            @Parameter(name = "client_version", description = "The Dockstore client version")
+            @Parameter(name = "client_version", description = "The Dockstore client version", schema = @Schema(pattern = "(^(\\d+)\\.(\\d+)\\.(\\d+)$)|(^development-build$)"))
```

Comment:
This could also be an enumeration of valid values, but then it would need to be updated with every release.